pizza.js

```diff
@@ -1,5 +1,6 @@
 OrderItems = new Mongo.Collection("order_items");
 Orders = new Mongo.Collection("orders");
+var timeToOrder = false;
```

Comment:
Variable name is a bit confusing, I thought it was a duration (10 minutes or so). Should perhaps be something like, hasOrdered or something similar indicating the boolean nature